package.json

```diff
@@ -62,9 +62,18 @@
     "@types/storybook__addon-storyshots": "^5.1.1",
     "@types/storybook__addon-viewport": "^4.1.1",
     "@types/storybook__react": "^4.0.2",
-    "eslint-config-prettier": "^6.2.0",
-    "eslint-plugin-prettier": "^3.0.1",
+    "@typescript-eslint/eslint-plugin": "^2.19.2",
+    "@typescript-eslint/parser": "^2.19.2",
+    "eslint": "6.8.0",
```

Comment:
All those explicit dependencies have been removed in the past because create react app pulls them in. With this change, you add them twice. What exactly was the idea/problem?

Reply:
I wanted to remove `eslint` from global requirements (globally installed package) to local. When I did this and tried to run `yarn lint` I got lot's of errors about missed packages.

Reply:
eslint is installed locally, see:

```
$ cd ponferrada
$ git checkout master
$ yarn install
$ ls -lA node_modules/.bin/eslint
lrwxr-xr-x  1 me  staff  50 17 Feb 16:03 node_modules/.bin/eslint -> ../addons-linter/node_modules/eslint/bin/eslint.js
$ ./node_modules/.bin/eslint --version
v5.16.0
$ yarn eslint --version
yarn run v1.22.0
$ /projects/ponferrada/node_modules/.bin/eslint --version
v5.16.0
✨  Done in 0.76s.
```

The only question for me is why this is not yet eslint 6 ....

Reply:
hmm. weird. Let me check again. I am always have troubles with `eslint` for some reason :(

Reply:
But uninstalling the global version is probably a good idea to avoid confusion

Reply:
Hmm, eslint 5 is installed via `mozilla/web-ext` and `mozilla/addons-linter`, which [still uses eslint 5](https://github.com/mozilla/addons-linter/pull/2652). But so far this did not make problems.